packages/flutter/test/foundation/change_notifier_test.dart

```diff
@@ -61,13 +61,15 @@ class Counter with ChangeNotifier {
 }
 
 void main() {
-  testWidgets('ChangeNotifier can not dispose in callback', (WidgetTester tester) async {
+  testWidgets('ChangeNotifier can not dispose in callback',
```

Comment:
Can you please undo the formatting changes in this file?

Reply:
done!